ClinicService.API/Controllers/DoctorController.cs

```diff
@@ -9,30 +9,46 @@ namespace Margarita_Grafa_PetProject2025.Controllers;
 public class DoctorController : ControllerBase
 {
     private readonly ILogger<DoctorController> _logger;
-    private readonly IDoctorRepository _doctorRepository;
+    private readonly IDoctorService _doctorService;
 
-
-    public DoctorController(ILogger<DoctorController> logger, IDoctorRepository doctorRepository)
+    public DoctorController(ILogger<DoctorController> logger, IDoctorService doctorServise)
     {
         _logger = logger;
-        _doctorRepository = doctorRepository;
+        _doctorService = doctorServise;
     }
 
     [HttpGet(Name = "GetDoctor")]
-    public async Task<DoctorEntity> Get(int id)
+    public async Task<DoctorModel> Get(Guid id)
     {
-        return new DoctorEntity();
+        var doctorModel = await _doctorService.GetById(id);
+
+        return doctorModel;
     }
 
     [HttpPost(Name = "PostDoctor")]
-    public async Task<DoctorEntity> Post(DoctorEntity item)
+    public async Task<DoctorModel> Post(DoctorModel item)
+
+    {
+        var doctor = new DoctorModel();
+        var response = await _doctorService.CreateAsync(doctor);
+
```

Comment:
Create method should consume Item from the controller params instead of that empty model